test/test.js

```diff
@@ -3,7 +3,7 @@ import test from 'ava';
 import 'babel-register';
 import 'babel-polyfill';
 
-import AnkiExport, { SEPARATOR } from '../src/index';
+import ankiExport, { SEPARATOR } from '../src/index';
```

Comment:
but passing additional params no has not value. Function-generator doesn't process them